app/src/main/java/com/serwylo/lexica/GameActivity.java

```diff
@@ -201,6 +201,7 @@ public void onResume() {
         if (game == null) {
             Toast.makeText(this, R.string.error_restoring_game, Toast.LENGTH_SHORT).show();;
             NavUtils.navigateUpFromSameTask(this);
+            return;
```

Comment:
Bonus fix, noticed while reading that if game is null, we will crash after this if statement unless we return.